app/controllers/posts_controller.rb

```diff
@@ -1,7 +1,7 @@
 class PostsController < ApplicationController
   def index
```

Comment:
- [ ] Hi @gvgesanto2, let's keep our application scalable by eliminating the N+1 query problems from our application(Seems n extra queries are being executed when fetching all posts and comments.) Kindly fix these issues.
- [ ] Kindly do the same for comments